### PR TITLE
chore: remove debug charts error

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

we used to care about super accurate balance charts but haven't for a long time as it's pushing shit up hill catching every edge case

this got broken in a recent refactor and is showing all the time. get rid of it.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk

basically none, we haven't cared about this forever anyway

### Engineering

just removing debug logging

### Operations

no testing required

## Screenshots (if applicable)

image

### CODE REVIEW

**Review by woodenfurniture — APPROVED** [review]
Lovely

**Review by 0xApotheosis — APPROVED** [review]
